packages/components/package.json

```diff
@@ -98,8 +98,11 @@
     "@react-aria/utils": "^3.27.0",
     "@react-spectrum/utils": "^3.12.1",
     "@storybook/test": "^8.6.12",
+    "aria": "^0.2.1",
```

Comment:
Just a last thing! It seems that this library is not used?

Reply:
removed :)